docs/vereisten/verantwoordingsplicht_rechtmatigheid.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Verantwoordingsplicht voor de rechtmatigheid van de verwerking
+toelichting: Bij het verwerken van persoonsgegevens voor algoritmes en AI-systemen moeten de verantwoordelijkheden kunnen aantonen dat de verwerking rechtmatig is.
+status_vereiste: 
+ - Geldend
+levenscyclus: 
+- ontwerp
+- dataverkenning-en-datapreparatie
+- ontwikkelen
+- monitoren
+- archiveren
+bouwblok: 
+- governance
+- privacy-en-gegevensbescherming
+---
+
+<!-- tags -->
+## Vereiste
+
+De verantwoordelijkheden moeten bij de verwerking van persoonsgegevens door algoritmes en AI-systemen kunnen aantonen dat de verwerkingen rechtmatig plaatsvinden.
+- Dit betekent concreet dat de volgende punten aangetoond kunnen worden:
+ - Rechtmatigheid, behoorlijkheid en transparantie.
+ - Doelbinding.
+ - Minimale gegevensverwerking.
+ - Juistheid.
+ - Opslagbeperking.
+ - Integriteit en vertrouwelijkheid
+Een aandachtpunt daarbij is dat de rechtmatigheid van de verwerking ten opzicht van andere gerelateerde wetgeving zoals de AI Act en de Algemene wet gelijke behandeling ook moeten kunnen worden aangetoond voor zover de rechtmatigheid van de verwerking onder de AVG daarvan afhankelijk is.
+
+## Toelichting 
+
+Bij het verwerken van persoonsgegevens voor algoritmes en AI-systemen moeten de verantwoordelijkheden duidelijk beschreven en toegewezen zijn.
+De verwerkingsverantwoordelijke is degene die ervoor zorgt dat deze verantwoordelijkheden worden nageleefd en kan dit aantonen, wat bekend staat als de verantwoordingsplicht.
+Deze maatregelen zijn essentieel om de naleving van regelgeving met betrekking tot gegevensbescherming te waarborgen en het vertrouwen van gebruikers in de verwerking van hun gegevens te vergroten.
+
+## Bronnen 
+
+| Bron                        |
+|-------------|
+|Artikel 5 AVG|
+
+## Wanneer van toepassing? 
+Deze vereiste is van toepassing op alle algoritmen die persoonsgegevens verwerken.
+
+
+## Risico 
```

Comment:
Goede punten bij risico, ik merk dat sommige zaken in mindere mate als risico zijn uitgewerkt en meer als toelichting. Ik heb een paar comments/voorstellen gedaan, maar in de basis moet dit uit elkaar worden gehouden.